invenio_assets/webpack.py

```diff
@@ -12,18 +12,50 @@
 from collections import OrderedDict
 
 from flask import current_app, request
-from flask_webpackext import WebpackBundle, WebpackBundleProject
+from flask_webpackext import WebpackBundle
+from flask_webpackext import WebpackBundleProject as WebpackBundleProjectBase
 from flask_webpackext.manifest import JinjaManifest, JinjaManifestLoader
+from invenio_base.utils import obj_or_import_string
 from markupsafe import Markup
 from pywebpack import ManifestEntry, UnsupportedExtensionError, bundles_from_entry_point
+from werkzeug.local import LocalProxy
 
-project = WebpackBundleProject(
+
+class WebpackBundleProject(WebpackBundleProjectBase):
+    """Flask webpack bundle project."""
+
+    def __init__(self, import_name, base_package_json="package.json", **kwargs):
+        """Constructor."""
+        super().__init__(import_name, **kwargs)
+        self._package_json_source_path = base_package_json
```

Comment:
this property can be set via the constructor in [`pywebpack.project:WebpackBundleProject`](https://github.com/inveniosoftware/pywebpack/blob/master/pywebpack/project.py#L188), but the constructor in [`flask_webpackext.project:WebpackBundleProject`](https://github.com/inveniosoftware/flask-webpackext/blob/master/flask_webpackext/project.py#L145) neither takes this value as an argument nor does it set it for the super constructor call.